client/my-sites/checkout/checkout/index.jsx

```diff
@@ -392,7 +393,10 @@ export class Checkout extends React.Component {
 		// - has a receipt number
 		// - does not have a receipt number but has an item in cart(as in the case of paying with a redirect payment type)
 		if ( selectedSiteSlug && ( ! isReceiptEmpty || ! isCartEmpty ) ) {
-			if ( isJetpackNotAtomic ) {
+			const isJetpackProduct = product && includes( JETPACK_BACKUP_PRODUCTS, product );
```

Comment:
Could we maybe use here the [`isJetpackProduct()` helper function](https://github.com/Automattic/wp-calypso/blob/master/client/lib/products-values/index.js#L204-L216)?

Reply:
No, because it accepts a `cartItem` as a parameter, and here we just use the slug. 

But it's fair to update `isJetpackProduct` to use the new constants, I'll file a PR for that. Thanks!